src/vue/components/inflections.vue

```diff
@@ -44,9 +44,13 @@
         <main-table-wide-vue :collapsed="false" :view="selectedView" @widthchange="updateWidth">
         </main-table-wide-vue>
 
-        <template v-for="linkedView in selectedView.linkedViews" v-if="selectedView.linkedViews">
-          <main-table-wide-vue :collapsed="false" :view="linkedView" @widthchange="updateWidth">
-          </main-table-wide-vue>
+        <template v-if="selectedView.linkedViews">
+          <main-table-wide-vue
+              :collapsed="false"
+              :view="linkedView"
+              @widthchange="updateWidth"
+              v-for="linkedView in selectedView.linkedViews"
```

Comment:
may be it is better to use v-key here, I think that linkedView is a rather complex object and it should be easier to track it dentity with a key? What do you think?

Reply:
Great idea, will add a `v-key` here.